### PR TITLE
Set up API docs through GitHub Pages

### PR DESCRIPTION
**Pull request by romanlutz**
This includes the basic docs setup & a GitHub workflow to deploy it to GitHub Pages after PRs are merged.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Responsible AI Widgets'
+copyright = 'Microsoft Corporation'
+author = 'Roman Lutz, Ilya Matiach, Ke Xu'
+
+# The full version, including alpha/beta/rc tags
+release = 'v0.2.0'
```
Comment (imatiach-msft) [inline docs/source/conf.py:30]: this is another place where we need to keep track of version and could reuse the same constant.  actually this should be 0.2.1 now.

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+html_theme = 'pydata_sphinx_theme'
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+html_theme_options = {
+    # TODO: fork the pydata-sphinx-theme to integrate these with logo
+    "external_links": [
+        # {"name": "StackOverflow",
+        #  "url": "https://stackoverflow.com/questions/tagged/fairlearn"}
```
Comment (imatiach-msft) [inline docs/source/conf.py:80]: remove fairlearn commented out stuff here?

File: raiwidgets/raiwidgets/explanation_dashboard.py [hunk 3]
```diff
@@ -45,7 +45,6 @@ class and for the regression case a method of predict()
     def __init__(self, explanation, model=None, dataset=None,
                  true_y=None, classes=None, features=None,
                  public_ip=None, port=None, locale=None):
-        """Initialize the fairness Dashboard."""
```
Comment (imatiach-msft) [inline raiwidgets/raiwidgets/explanation_dashboard.py:48]: are you sure we should remove such comments?  maybe just update to say explanation dashboard?

**Review by imatiach-msft — APPROVED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (romanlutz) [reply docs/source/conf.py:30]: Ah, here I should be able to just reference the other one.

File: docs/source/conf.py (hunk 2 above)
Comment (romanlutz) [reply docs/source/conf.py:80]: I want to eventually add StackOverflow, but currently there's no tag for raiwidgets and I don't have permissions to add such a tag on StackOverflow :-) I'll change the tag to raiwidgets in the text, though.

File: raiwidgets/raiwidgets/explanation_dashboard.py (hunk 3 above)
Comment (romanlutz) [reply raiwidgets/raiwidgets/explanation_dashboard.py:48]: Do they add anything? The method is called `__init__` in a class called `ExplanationDashboard`. I'll add it back, but I'm typically in favor of useful comments only.

**Pull request merged**